RecoMET/METAlgorithms/src/METSignificance.cc

```diff
@@ -20,16 +20,18 @@ Description: [one line class summary]
 
 metsig::METSignificance::METSignificance(const edm::ParameterSet& iConfig) {
 
-  edm::ParameterSet cfgParams = iConfig.getParameter<edm::ParameterSet>("parameters");
+  // if(iConfig.exists("parameters")) {
```

Comment:
not needed anymore, forgot to remove it during the development period, going to be removed